test.sh

```diff
@@ -165,4 +247,13 @@ function runHTTPSTests() {
   ${conjur_test_cid} ${tests_command}
 }
 
+#function finish {
+#  echo '-----------------------test.sh------------------------------'
+#  echo 'Removing test environment'
+#  echo '------------------------------------------------------------'
+#  docker-compose down -v
+#}
+
 main
+
+trap finish EXIT
```

Comment:
You want this moved higher in the file, and especially before `main`. If you have this command here, you will not trap any events from failures in `main` which is what this command is designed to do.

Reply:
👍